src/components/Dashboard/Dashboard.js

```diff
@@ -1,10 +1,9 @@
 /* eslint-disable prettier/prettier */
-
 import React from 'react'
 import { Header, Split } from '@aragon/ui'
 import ProfileHeader from './ProfileHeader'
 import DashboardStats from './DashboardStats'
-import Import from './Import'
+// import Authorize from './Authorize'
 // import TaskTable from './TaskTable'
```

Comment:
Remove commented imports if not required

Reply:
Removed